admin_manual/installation/nginx.rst

```diff
@@ -90,8 +90,13 @@ webroot of your nginx installation. In this example it is
       client_max_body_size 512M;
       fastcgi_buffers 64 4K;
   
-      # Disable gzip to avoid the removal of the ETag header
-      gzip off;
+      # Enable gzip but do not remove ETag headers
+      gzip on;
+      gzip_vary on;
```

Comment:
@josh4trunks Forgot to ask you: Any opinion on this? Works here fine and seems to make sense from my point of view.